README.md

```diff
@@ -117,8 +117,8 @@ JOIN OUR MEETINGS REGULARLY OR CONTRIBUTE, THEN ADD YOURSELF VIA PULL REQUEST
 <details><summary>Click to view list</summary>
 
 * Michael Michael ([@michmike](https://github.com/michmike)), VMware
-* Tom Kerkhove ([@tomkerkhove](https://github.com/)), Codit
-* Jeff Hollan ([@jeffhollan](https://github.com/jeffhollan)), Microsoft
+* Tom Kerkhove ([@tomkerkhove](https://github.com/)), Microsoft
```

Comment:
fix url please to not point on github's frontpage.

Reply:
Well spotted
```suggestion
* Tom Kerkhove ([@tomkerkhove](https://github.com/tomkerkhove)), Microsoft
```